### PR TITLE
Add Plist Plugin

### PR DESCRIPTION
**Pull request by weslambert**

### CODE REVIEW

File: vql/parsers/plist.go [hunk 1]
```diff
@@ -57,14 +79,86 @@ func (self *PlistParser) Call(ctx context.Context,
...
+
+				file, err := accessor.Open(filename)
+				if err != nil {
+					scope.Log("Unable to open file %s: %v",
+						filename, err)
+					return
+				}
+				
+				defer file.Close()
+
```
Comment (scudette) [inline vql/parsers/plist.go:123]: Typically Go projects run the code through gofmt to get the spacing and formatting right.

File: vql/parsers/plist.go [hunk 2]
```diff
@@ -2,25 +2,47 @@ package parsers
...
 	Accessor string `vfilter:"optional,field=accessor,doc=The accessor to use."`
 }
 
-type PlistParser struct{}
+type PlistFunction struct{}
+
+type _PlistPlugin struct{}
 
-func (self *PlistParser) Call(ctx context.Context,
+func (self *PlistFunction) Call(ctx context.Context,
```
Comment (scudette) [inline vql/parsers/plist.go:42]: Can you please rearrange so the methods go together with the object? it is a bit easier to follow when all related methods are together.

**Review by scudette — CHANGES_REQUESTED** [review]

File: vql/parsers/plist.go (hunk 1 above)
Comment (weslambert) [reply vql/parsers/plist.go:136]: Okay, I've used `gofmt` to format `plist.go` and added another commit.

File: vql/parsers/plist.go (hunk 2 above)
Comment (weslambert) [reply vql/parsers/plist.go:43]: I've re-arranged it a bit and added another commit.  Let me know if that looks in line with what you are expecting.

**Comment by scudette** [comment]
Thanks!

**Pull request merged**